openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -145,16 +146,18 @@ public void pluginInit(StfCoreExtension test) throws Exception {
 		StfTestArguments testArgs = test.env().getTestProperties("tests","jckversion","testsuite","config=[NULL]","executiontype=[multijvm]","withagent=[off]","interactive=[no]","jckRoot=[NULL]","concurrency=[NULL]");
 		
 		testJdk = System.getenv("JAVA_HOME");
-		tests = testArgs.get("tests");
+		tests = testArgs.get("tests");	
+		jckVersion = testArgs.get("jckversion");	
+		jckVersionNo = jckVersion.replace("jck", "");		
+		jckUnzippedFolderName = "JCK" + jckVersionNo + "-unzipped"; 
```

Comment:
Same here. The folder name should be passed in.

Reply:
I believe this is different than the SRCDIR and OUTDIR in the build files. This JCK main folder name is being constructed only using JCK version, which is being passed via the JCK Playlist command linens where Jck.java is invoked. Whoever is running this class should have JCK staged ahead of time, which means the JCK-<jckVersionnNo-unzipped folder would be available. So, I am not sure if we really have to update all the playlists and pass in the unzipped folder name.

Reply:
Talked with Mesbah, the main idea is to allow users to use the script if they have a different jck repo/dir.

Reply:
Hi Lan, changes have been made to the PR. Please check. (sent grinder links to your slack).